backend/src/main/scala/com/softwaremill/bootzooka/util/DefaultClock.scala

```diff
@@ -0,0 +1,20 @@
+package com.softwaremill.bootzooka.util
+
+import java.time.Instant
+
+import cats.effect.Clock
```

Comment:
If we rename to `Clock`, you can then shadow this import e.g. `import cats.effect.{Clock => CatsClock}` to avoid name conflicts

Reply:
Very elegant, thanks